lib/onelogin/ruby-saml/response.rb

```diff
@@ -200,27 +487,58 @@ def xpath_first_from_signed_assertion(subelt=nil)
         node
       end
 
+      # Extracts all the appearances that matchs the subelt (pattern)
+      # Search on any Assertion that is signed, or has a Response parent signed
+      # @param subelt [String] The XPath pattern
+      # @return [Array of REXML::Element] Return all matches
+      #
+      def xpath_from_signed_assertion(subelt=nil)
+        node = REXML::XPath.match(
+            document,
```

Comment:
This is a more readable style than most other spots touched in this changeset. For consistency, however, please either change this to be one long line, or update the other spots to span multiple lines.